Task_TheGridProblem/index.css

```diff
@@ -1 +1,38 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */
+*{
+    margin: 0;
+    padding: 0;
+}
+body{
+    background-color: aquamarine;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 60px;
+    
+
+}
+.grid-container{
+    display: grid;
+    grid-template-columns: repeat(3,100px);
+    grid-gap: 3px;
+    background-color: blue;
+    border-radius: 30px;
+    padding: 40px;
+    width: 23%;

```

Comment:
There are multiple empty lines between line 22 and line 24, these should be removed.